packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -7,16 +7,48 @@ import Color from "@khanacademy/wonder-blocks-color";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
 
-export default class TextField extends React.Component<{||}> {
+type State = {|
+    enabled: boolean,
```

Comment:
In this case it's better to have only one variable that holds the `enabled/disabled` state. There could be situations for X reasons where both could be `true` and that would lead to an invalid state. In that case it's better to get rid of `enabled` and use only `disabled=true|false` instead.

Reply:
Yes. That makes total sense. I was a bit confused on why the spec had both when I started to code this up. I went too deep into this and just thought that I would have to internally ensure that `enabled` and `disabled` were never true at the same time. This clears it up 😄

Reply:
yeah no worries, sometimes there will be some stuff that doesn't translate exactly from design to UI :). And I think that's one of the reasons why we use implementation specs, so we can create a bridge between Design and Code.